src/static/js/chapter.js

```diff
@@ -1,3 +1,61 @@
+//This function removes the lazy-loading attributes from all img and iframe tags
+//Useful for print view for example (https://bugs.chromium.org/p/chromium/issues/detail?id=875403)
+function removeLazyLoading() {
+
+  console.log("Removing lazy loading...");
+
+  var all_imgs = document.querySelectorAll('img');

```

Comment:
Combine img and iframe into one query, and limit to elements with loading attrs. You could also use forEach to simplify the loop.
```suggestion
  Array.from(document.querySelectorAll('img[loading], iframe[loading]')).forEach(...);
```

Reply:
IE 11 doesn’t like `foreach` so have tried to avoid that.

Reply:
I think it's supported https://caniuse.com/#feat=mdn-javascript_builtins_array_foreach

Reply:
Ah maybe. Will give it a go. Did try it for something else and that’s when I notice it failed but that might have been using this: https://caniuse.com/#feat=mdn-api_nodelist_foreach

Reply:
Isn't supported in IE11. But then neither is native-lazy loading so doesn't matter here. And code is MUCH cleaner - so thanks!